pandas/tests/window/test_pairwise.py

```diff
@@ -99,6 +99,7 @@ def test_flex_binary_frame(method, frame):
     exp = DataFrame(
         {k: getattr(frame[k].rolling(window=10), method)(frame2[k]) for k in frame}
     )
+    exp.columns = exp.columns.astype(np.int_)
```

Comment:
can this be np.int64?

Reply:
ditto elsewhere.  in general i dont think we use `np.int_` often

Reply:
maybe np.intp?

Reply:
It can be np.int64 here, I've updated the fixture. I've also updated the whole commit to use np.intp.